docs: add new Layout Functions: Center, Column and Row.. (#516)
ZupIT/beagle-docs: content/en/api/components/layout/row.md

---
title: Row
weight: 338
description: You will find here the description of the Row function and its attributes details
---

---

## What is it?

> `Row` is a function that displays your children in a horizontal matrix.

Reviewer: ```suggestion
`Column` is a function to help you organize the layout of your view. It  creates a `Container` that displays its content in a horizontal line. This is equivalent to create a `Container` with `flexDirection` equal to `ROW`.
```

See how the structure is represented:

| **Attribute** | **Type**  | Required | **Definition** |
| :----------- | :------------------------------------------------------------- | :---------: | :---------------------------------------------------------------------------------------------------------------- |

> | children     | List &lt;[ServerDriven Component]({{< ref path="/api/components" lang="pt" >}})&gt; |            | The components below this component in the tree.

Reviewer: ```suggestion
| children     | List &lt;[ServerDriven Component]({{< ref path="/api/components" lang="pt" >}})&gt; |            | The content of this Container, i.e. its children.
```

> | context      | [ContextData]({{< ref path="/api/context/" lang="pt" >}})                           |             | It is the [**context**]({{< ref path="/api/context" lang="en" >}}) in this widget.                                                    |

Reviewer: ```suggestion
| context      | [ContextData]({{< ref path="/api/context/" lang="pt" >}})                           |             | The [**context**]({{< ref path="/api/context" lang="en" >}}) linked to this widget.                                                    |
```
